server/repository/src/mock/mod.rs

```diff
@@ -1735,3 +1736,62 @@ impl MockData {
         self
     }
 }
+
+pub fn make_movements(stock_line: StockLineRow, date_quantity: Vec<(i64, i64)>) -> MockData {
```

Comment:
Just wondering if we should move this back to test_helpers? Though not aware of context as to why it got moved here